src/amo/fonts/inter.scss

```diff
@@ -0,0 +1,100 @@
+@font-face {
```

Comment:
I notice that we include some `woff2` and `otf` files, e.g. `Inter-ExtraBold` and `Inter-Black` for which there are no definitions in this file. Does that mean those fonts aren't used anywhere on the site, and if so, should we exclude those files? Or does it not really matter than the files are in the repo if they're never referenced in this file?

Reply:
I don't think it matters if they're never referenced.  I included everything in case we were to reference them in the future, but perhaps in the name of tidiness they shouldn't be there?

Reply:
https://github.com/mozilla/addons-frontend/pull/10110/commits/7ff730835fb6130a98bb8fec622dd52637dcabf2